src/flightlog.js

```diff
@@ -519,9 +524,9 @@ export function FlightLog(logData) {
                 }
                 break;
               case "H":
-              // TODO
-              // contains coordinates only
-              // should be handled separately
+                const homeAltitude = frame.length > 2 ? frame[2 ] /10 : 0; // will work after BF firmware improvement
```

Comment:
need to declare with let before the switch case and assign value here.

Reply:
What result will ins this variable with different frames type? We need it for "H" type only.
The SonarCloud shows the issue, but in my opinion, this is more good place to define it.
I've tryed to put this expression in the next code rows directly, without variable. But SonarCloud shows the issue too.

Reply:
Sonar won't complain if you wrap the case in brackets.

Reply:
Yes, it works.